### PR TITLE
MM-17515 - Updating settings-min css

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
MM-17515 - Updating settings-min css

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-17515

### CODE REVIEW

File: sass/routes/_settings.scss [hunk 1]
```diff
@@ -653,3 +608,51 @@
...
+        margin-right: 5px;
+    }
+}
+
+.section-min__describe {
+    @include clearfix;
+    @include opacity(.7);
+    text-overflow: ellipsis;
+    white-space: pre;
+}
```
Comment (marianunez) [inline sass/routes/_settings.scss]: Missing new line

**Comment by lindy65** [comment]
Thanks @asaadmahmood 

Tested and it looks good to me.

Just one small nit (not sure if you missed it in the ticket?) but there is a period after `Click to upload an image.` on Profile picture and none of the other options have periods afterwards.

Are you able to remove the period?

image

Otherwise looks good. Confirmed that unit tests are done too.

QA will re-review/test post merge.

**Comment by asaadmahmood** [comment]
Updated the translation.
Screenshot 2019-08-08 at 12 59 16 PM

**Comment by marianunez** [comment]
@asaadmahmood I don't see the text change in the committed files. Is that in another PR?

**Comment by asaadmahmood** [comment]
Hmmm good point, not sure if I forgot to push.
I'll push it here: https://github.com/mattermost/mattermost-webapp/pull/3390
As it was not strictly related to this PR anyways.

**Pull request merged**